infra/aws/terraform/prow-build-cluster/terraform.canary.tfvars

```diff
@@ -15,14 +15,13 @@ limitations under the License.
 */
 
 aws_account_id = "054318140392"
-eks_admins = [
+
+eks_viewers = [
```

Comment:
We're still users with admin permissions (and we'll still need those users), so I think we should call this `eks_admins` or something like that.

Reply:
But this list only allows us to assume viewer role. Other then that we are still admins but only when assume different role.

Reply:
Okay, I see. :+1: